pkg/controller/constraint/constraint_controller.go

```diff
@@ -65,9 +65,12 @@ import (
 
 const (
 	BlockVAPBGenerationUntilAnnotation = "gatekeeper.sh/block-vapb-generation-until"
+	VAPBGenerationAnnotation           = "gatekeeper.sh/vapb-generation"
```

Comment:
nit: `vapb-generation-lock` or similar to more clearly specify intent

Reply:
How about `vapb-generation-state`?